src/main/java/seedu/taassist/logic/parser/SessionCommandParser.java

```diff
@@ -0,0 +1,37 @@
+package seedu.taassist.logic.parser;
+
+import static seedu.taassist.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static seedu.taassist.logic.parser.CliSyntax.PREFIX_SESSION;
+
+import seedu.taassist.logic.commands.SessionCommand;
+import seedu.taassist.logic.parser.exceptions.ParseException;
+import seedu.taassist.model.session.Session;
+
+/**
+ * Parses input arguments and creates a new SessionCommand object
+ */
+public class SessionCommandParser implements Parser<SessionCommand> {
+    /**
+     * Parses the given {@code String} of arguments in the context of the SessionCommand
+     * and returns an AssignCommand object for execution.
+     * @throws ParseException if the user input does not conform the expected format
```

Comment:
```suggestion
     * @throws ParseException if the user input does not conform the expected format.
```
I realise that even though some code given does not have the full stop, but the java coding standard  has full stop for every line of javadoc, does this matter?

Reply:
I think we will have to rephrase some sentences. AB3 tend to have their @throws or @param descriptions starting with a lowercase, while the style guide use complete sentences and start with an uppercase, that's why they all contain fullstops. We might need to take a general review of all Javadoc comments to standardize them.